api/tests/routes/backoffice/offerers_test.py

```diff
@@ -2302,7 +2283,7 @@ def test_list_filtering_by_date(self, authenticated_client):
         rows = html_parser.extract_table_rows(response.data)
         assert [int(row["ID Compte pro"]) for row in rows] == [uo.user.id for uo in (user_offerer_3, user_offerer_2)]
 
-    @pytest.mark.parametrize("postal_code", ["97100", "97 100"])
+    @pytest.mark.parametrize("postal_code", ["97200", "97 200"])
```

Comment:
Pourquoi ce changement ? b@example.com a bien pour code postal 97100
```suggestion
    @pytest.mark.parametrize("postal_code", ["97100", "97 100"])
```

Reply:
En fait pour éviter de filtrer en passant PENDING, tu pouvais laisser 97200, mais il fallait a@example.com en adresse attendue. Bref, modifier de manière cohérente :)